src/Schema/Table.php

```diff
@@ -346,6 +351,48 @@ public function addColumn($name, $typeName, array $options = [])
         return $column;
     }
 
+    /** @return array<string, string> */
+    final public function getRenamedColumns(): array
+    {
+        return $this->renamedColumns;
+    }
+
+    /**
+     * @throws LogicException
+     * @throws SchemaException
```

Comment:
@Tofandel where in this method is `SchemaException` thrown?

Reply:
It's not, i think it was a leftover because I was not sure which exception to chose and I started with SchemaException

Reply:
@Tofandel do you mind filing a PR that removes it? PhpStorm treats it as check exception meaning that the caller of this method should either catch it or have a `@throws SchemaException` in its declaration. This is false-positive and should be fixed.